app.js

```diff
@@ -199,6 +200,31 @@ app.get('/favicon.ico', (req, res) => {
   res.end();
 });
 
+app.delete('/api/:locationId/location', (req,res) =>{
+  const
+    locationId = req.params.locationId;
+
+  db.removeLocation(locationId)
+    .then(() => res.send().end())
+    .catch(error => {
+      log.info(error);
+      res.status(500).end('server error!');
+    });
+});
+
+app.post('/api/:userId/:locationId/updateLocation', (req,res) =>{
+  const {locationName, country, city, postal} = req.body,
+    userId = req.params.userId,
+    locationId = req.params.locationId;
```

Comment:
You could use destructuring on `userId` and `locationId` as well.  Something like:
```javascript
const { locationName, country, city, postal } = req.body;
const { userId, locationId } = req.params;
```